{{cookiecutter.project_slug}}/requirements.test.txt

```diff
@@ -1,5 +1,7 @@
+-e git+https://github.com/jssuzanne/anyblok@mysql_and_pytest#egg=anyblok
```

Comment:
self reminder, remove this line before merging (after pytest branch merged in anyblok project)